articles/tutorials/building_2d_games/16_service_container/index.md

```diff
@@ -0,0 +1,482 @@
+---
+title: "Chapter 16: The MonoGame Services Container"
+description: "Learn how to use MonoGame's Service Container to create a modular game architecture, enabling components to easily find and communicate with each other."
+---
+
+In [Chapter 14](../14_soundeffects_and_music/index.md), we added audio feedback to our game by implementing sound effects and background music. These audio features, along with our input handling from earlier chapters, are implemented as reusable game components.
+
+As our game grows more complex, we'll need to manage numerous systems and share resources across different parts of code.  Currently, we are storing direct references to our components, which creates tight coupling between our game and the specific implementations of these systems.  In the following chapters, we are going to start breaking the game up into different modules and will need to share resources between them.  To do this, a more flexible approach would be use to use MonoGame's built-in service system to make these components available throughout the game.
```

Comment:
Probably worth a note, that this is a more advanced chapter and is optional?  Just to not put off people who "don't get it"

But highly recommended :D